api/base/parsers.py

```diff
@@ -25,14 +50,31 @@ def parse(self, stream, media_type=None, parser_context=None):
         data = result.get('data', {})
 
         if data:
-            if 'attributes' not in data:
-                raise JSONAPIException(source={'pointer': '/data/attributes'}, detail=NO_ATTRIBUTES_ERROR)
+            relationships = data.get('relationships')
+
+            path = stream.path
+            related_resources = ['contributors', 'node_links']
```

Comment:
Okay, instead of hardcoding the items for which I expect to receive a "relationships" object, I am adding is_relationship = True to the parser_context.